src/Sulu/Bundle/ContactBundle/Resources/config/lists/contacts.xml

```diff
@@ -296,12 +296,13 @@
         </property>
 
         <group-concat-property
-            name="tagIds"
-            visibility="never"
```

Comment:
This tagIds should stay as the were else it would be a backward compatibility break.

Create an additional tagNames group concat property instead.